samples/sample_common/include/vaapi_utils.h

```diff
@@ -430,6 +435,18 @@ namespace MfxLoader
 
 } // namespace MfxLoader
 
+typedef struct _SurfaceInfo
```

Comment:
Please use just struct. Typedef is only reasonable for shared C/C++ headers. But here we have C++-only code.